Config/routing.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+
+<routes xmlns="http://symfony.com/schema/routing"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://symfony.com/schema/routing http://symfony.com/schema/routing/routing-1.0.xsd">
+
+
+    <route id="AbandonedCart.admin.configure" path="/admin/module/AbandonedCartReminder/configure" methods="post">
```

Comment:
Keep lower case for route ids and AbandonedCart is not the full name of the module, should be `abandoned_cart_reminder.admin.configure`